src/main.rs

```diff
@@ -209,12 +206,16 @@ async fn handle_ldk_events(
 			network_update: _,
 			all_paths_failed,
 			path: _,
+			short_channel_id,
 		} => {
 			print!(
 				"\nEVENT: Failed to send payment{} to payment hash {:?}: ",
```

Comment:
The trailing colon needs to go after the short_channel_id message below, if included.